scripts/common.py

```diff
@@ -216,6 +216,7 @@
     'blog-editor-tests/try-to-publish-a-duplicate-blog-post-and-get-blocked',
     'exploration-editor-tests/create-exploration-and-change-basic-settings',
     'exploration-editor-tests/load-complete-and-restart-exploration-preview',
+    'exploration-editor-tests/savedraft-publish-and-discard-the-changes',
```

Comment:
Reminder to update filename here as well.

Reply:
done